config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -37,21 +37,24 @@ spec:
       name: hardwareevents.event.redhat-cne.org
       version: v1alpha1
   description: |
+    ## Funtionality
+    Install and manage a hardware event delivery framework.
```

Comment:
You should start with CR I guess
To start using Hardware event proxy, install the operator and then create the following HardwareEvent:

  apiVersion: "event.redhat-cne.org/v1alpha1",
  kind: "HardwareEvent",
  metadata:
    name: "hardwareevent"
    spec:
      transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
Explain what is transport host.

Reply:
done